src/components/map/MapService.js

```diff
@@ -935,15 +940,19 @@ goog.require('ga_urlutils_service');
           return $http.get(url).then(function(response) {
             // Live modifications for 3d test
             if (response.data) {
+              // pngjpeg is not supported by zeitreihen in 3D
               var ids = [
-                'ch.swisstopo.swissimage-product',
-                'ch.swisstopo.pixelkarte-farbe',
-                'ch.swisstopo.pixelkarte-grau',
                 'ch.swisstopo.zeitreihen'
               ];
               angular.forEach(ids, function(id) {
                 if (response.data[id]) {
-                  response.data[id].config3d = id + '_3d';
+                  var config3dId = id + '_3d';
+                  var config3d = angular.extend({},
+                      response.data[id], config3d);
+                  config3d.format = 'jpeg';
+                  config3d.useMapProxy = true;
+                  response.data[config3dId] = config3d;
+                  response.data[id].config3d = config3dId;
```

Comment:
I think this discussion join the discussion we have further. The goal of the inheritance of the config3d is exactly the possibility to change only one property (format, attribution, subLayersId timestamp) and not duplicate all.